Automattic/wp-calypso
# Add Jetpack Plugin Review Prompt to Scan flow

#### Changes proposed in this Pull Request

* Adds a review prompt to the main Scan page, only if the user has completed a successful threat auto-fix.

<img width="742" alt="Screenshot on 2021-03-23 at 12-04-13" src="https://user-images.githubusercontent.com/11078128/112182203-77d8f680-8bd3-11eb-8248-61bae468ea8b.png">


#### Testing instructions

- checkout and run this PR in Jetpack Cloud ('yarn start-jetpack-cloud`)
- Select a site that has Jetpack Scan enabled.
- Go to the `/wp-content/plugins/` directory and modify any main plugin file and add the line:
    - `$eicar = "X5O!P%@AP[4\PZX54(P^)7CC)7}$EICAR-STANDARD-ANTIVIRUS-TEST-FILE!$H+H*";`
    - ( this will trigger a scan threat that Jetpack can auto-fix in the next steps)
- Now go to `http://jetpack.cloud.localhost:3000/scan/:site`
- Click "Scan now" button to start a scan.
- When scan finishes (you should have 1 scan threat), click "Auto fix 1 threat", and confirm "Fix all threats" button. 
- When auto-fix finishes, you should see the review prompt at the bottom of the Scan card like shown in the image above.
- Refresh the page and verify that the review prompt is persistent.
- Click the "Leave Jetpack a review" button and verify that it open the Jetpack plugin review page and that the review prompt disappears. 

Related to # 1192874419362946-as-1199999287485857
